src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/jpa/entity/DefendantAliasEntity.kt

```diff
@@ -0,0 +1,38 @@
+package uk.gov.justice.digital.hmpps.personrecord.jpa.entity
+
+import jakarta.persistence.CascadeType
+import jakarta.persistence.Column
+import jakarta.persistence.Entity
+import jakarta.persistence.GeneratedValue
+import jakarta.persistence.GenerationType
+import jakarta.persistence.Id
+import jakarta.persistence.JoinColumn
+import jakarta.persistence.ManyToOne
+import jakarta.persistence.Table
+import org.hibernate.envers.Audited
+
+@Entity
+@Table(name = "defendant_alias")
+@Audited
+class DefendantAliasEntity(
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long? = null,
+
+  @ManyToOne(optional = false, cascade = [CascadeType.ALL])
+  @JoinColumn(
+    name = "fk_defendant_id",
+    referencedColumnName = "id",
+    nullable = false,
+  )
+  var defendant: DefendantEntity? = null,
+
+  @Column(name = "first_name")
+  val firstName: String? = null,
+
+  @Column(name = "middle_name")
+  val middleName: String? = null,
+
+  @Column(name = "last_name")
+  val lastName: String? = null,
```

Comment:
As per address, is there anything to be said for simply having a "name" field? What if their name is Cristiano Ronaldo dos Santos Aveiro for example?

Reply:
But if we do decide to have it split, let us be consistent. We have a `surname` on defendant and `lastName` on defendant_alias

Reply:
Raised this last week as well, We have decided to have a chat and decide later on.

Reply:
I am going keep the surname on both tables. if they want to to change will change latter.